README.md

```diff
@@ -170,6 +166,90 @@ This will create a `closeAndReleaseRepository` task that you can call after `pub
 
 It assumes there's only one staging repository active when closeAndReleaseRepository is called. If you have stale staging repositories, you'll have to delete them by logging at https://oss.sonatype.org (or you Nexus instance).
 
+# Base plugin
+
+Starting with version 0.15.0 there is a base plugin. This new plugin has the same capabilities as the main 
+plugin but does not configure anything automatically. In the current stage the APIs are still marked with `@Incubating`
+so they might change.
+
+In your root `build.gradle` file you can do the general configuration for all modules in your project.
+
+```groovy
+import com.vanniktech.maven.publish.SonatypeHost
+
+allprojects {
+    plugins.withId("com.vanniktech.maven.publish.base") {
+        GROUP = "com.example.project"
+        VERSION = "1.0.3-SNAPSHOT"
+        
+        mavenPublishing {
+            // alternatively "S01"
+            publishToMavenCentral("DEFAULT")
```

Comment:
Can we get?

```suggestion
            publishToMavenCentral()
```

Reply:
It doesn't have a default value right now. The reason why I didn't in the base plugin API is that since you have to call this anyways I though it's nicer to have it explicit. More and more users will have s01. No strong opinion though